infrastructure/tooling/templates/k8s/hpa.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: ${projectName}-hpa
+  labels: {$eval: labels}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: ${projectName}
+  minReplicas: ${minReplicas}
```

Comment:
I think we need to treat this file a bit differently.. I know json-e and helm interop might be confusing :) 

So when we run it this way, json-e treats them as strings, see generated HPAs above - they all are quoted `'x'`
and if you run it like this we get:

```
Error from server (BadRequest): error when creating "k8s.yml": HorizontalPodAutoscaler in version "v2" cannot be handled as a HorizontalPodAutoscaler: json: cannot unmarshal string into Go struct field HorizontalPodAutoscalerSpec.spec.maxReplicas of type int32
```

probably json-e is not the right tool here, but that's another discussion :)

I think we can use something similar to what `REPLICA_CONFIG_STRING` is doing in those substitutions

Reply:
I couldn't push to this branch for some reason, maybe you can cherry-pick or just apply this fix:

https://github.com/taskcluster/taskcluster/pull/7718/commits/0f35c0943de7e797cb415c1bb69fb97d50f548af
https://github.com/taskcluster/taskcluster/pull/7718/commits/f11ac2abbc313181b239b16eb166e557438d4698